src/python/pants/bin/pants_runner.py

```diff
@@ -57,9 +57,9 @@ def _should_run_with_pantsd(self, global_bootstrap_options: OptionValueContainer
 
     @staticmethod
     def scrub_pythonpath() -> None:
-        # Do not propagate any PYTHONPATH that happens to have been set in our environment
+        # Don't propagate any PYTHONPATH that happens to have been set in our environment
         # to our subprocesses.
-        # Note that don't warn (but still scrub) if RUNNING_PANTS_FROM_SOURCES is set. This allows
+        # NB: We don't warn (but we still scrub) if RUNNING_PANTS_FROM_SOURCES is set. This allows
```

Comment:
Again, too nitpicky for my taste. I'm fine merging your changes, but in future I don't think this is better.